docs/source/federated-types/sharing-types.mdx

```diff
@@ -0,0 +1,404 @@
+---
+title: Sharing types and fields
+sidebar_title: Sharing types and fields
+description: Sharing types and fields across multiple subgraphs
+---
+
+import { CodeColumns } from 'gatsby-theme-apollo-docs';
+
+> ⚠️ The field sharing model has changed significantly in Federation 2. For a summary of these changes, [see what's new](/federation-2/new-in-federation-2/#value-types). 
+>
+>For more information about using value types in Federation 1, you can view the [previous version of this article here](https://www.apollographql.com/docs/federation/value-types/).
+
+In a federated graph, it's common to want to reuse a GraphQL type in multiple [subgraphs](../subgraphs/). 
+
+For example, suppose you want to define and reuse a generic `Position` type in different subgraphs:
+
+```graphql
+type Position {
+  x: Int!
+  y: Int!
+}
+```
+
+This article describes how to share a [GraphQL object type's](https://www.apollographql.com/docs/apollo-server/schema/schema/#object-types) fields in a federated graph, enabling multiple subgraphs to define and resolve those shared fields. 
+
+## Using `@shareable`
```

Comment:
I think we can deduplicate some content in this article and end up with the following header structure:

* Sharing object types
    * Varying shared fields
        * Differing return types
        * Omitting fields
* Enums, unions, and interfaces

All the stuff currently under "Using `@shareable`" can go under "Sharing object types". The subgraph examples in those two existing sections have a lot of overlap, so I think they'll merge nicely. Let me know if that doesn't sound correct!